.travis/build-site.sh

```diff
@@ -13,21 +13,9 @@ fi
 
 
 (
-    export PING_SLEEP=30s
-    export BUILD_OUTPUT=/tmp/build-site.out
-    export PING_PID_FILE=/tmp/build-site-ping.pid
-
-    source .travis/background-job-funcs.sh
-
     # Run the build, redirect output into the file
-    ./mvnw install -DskipTests=true -B -V >> $BUILD_OUTPUT 2>&1
-    ./mvnw site site:stage -Psite -B -V >> $BUILD_OUTPUT 2>&1
-
-    # The build finished without returning an error so dump a tail of the output
-    dump_output
-
-    # nicely terminate the ping output loop
-    kill_ping
+    travis_wait ./mvnw install -DskipTests=true -B -V
```

Comment:
I explicitly added this, since travis stopped the [build 711](https://travis-ci.org/pmd/pmd/builds/205242398). The last lines of the [raw-log](https://s3.amazonaws.com/archive.travis-ci.org/jobs/205242399/log.txt) read:

~~~
The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).

The job has been terminated
~~~

Not sure, if this limit is still enforced...